lib/openc_bot/helpers/incremental_search.rb

```diff
@@ -20,6 +20,46 @@ def fetch_data_via_incremental_search(options = {})
         new_highest_numbers
       end
 
+      def fetch_gaps
+        $stderr.puts "#{Time.now.utc.iso8601} - Starting fetch_gaps"
+        current_number = nil 
+        gap_count = 0
+    
+        min_company_number = get_var("minimum_company_number")
```

Comment:
We don't really need to save the min_company_number to the database.
When there is no highest_gap_uid, then query the database